sdk/eventhub/event-hubs/src/eventHubConsumerClientModels.ts

```diff
@@ -1,4 +1,4 @@
-import { CloseReason } from "./eventProcessor";
+import { CloseReason } from "./models.public";
```

Comment:
Nit: can we use `models/public` and `models/private`? (dashes would be fine too). I don't think I've come across using a period in a module path before in node.js (unless specifying the extension), so it feels out of place to me.

Reply:
Absolutely!